packages/sign-client/test/lifecycle/lifecycle.spec.ts

```diff
@@ -12,7 +12,7 @@ import { describe, it, expect, afterEach } from "vitest";
 
 const environment = process.env.ENVIRONMENT || "dev";
 
-const timeout = environment === "prod" ? 610_000 : 70_000;
+const timeout = environment === "prod" ? 610_000 : 30_000;
```

Comment:
feel free to link this against the server config https://github.com/WalletConnect/rs-relay/blob/master/terraform/vars/staging.tfvars#L9-L11